pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/OpChainSchedulerService.java

```diff
@@ -160,8 +155,8 @@ public final void register(OpChain operatorChain, boolean isNew) {
           operatorChain, isNew, _scheduler.size());
 
       _scheduler.register(operatorChain, isNew);
-      operatorChain.getStats().queued();
     } finally {
+      operatorChain.getStats().queued();
```

Comment:
can we add a test for this and the change above? I wasn't sure i fully understand why this queued call needs to be moved into the finally block

Reply:
This is just stats. it doesn't have any impact. I can move it back if needed.

Reply:
please do